pages/k8s/using-vault.md

```diff
@@ -83,44 +84,66 @@ exit
 juju run-action vault/0 authorize-charm token={charm token}
 ```
 
-Note that it is _critical_ that you save all five unseal keys as well as the
-root token.  If the **Vault** unit is ever rebooted, you will have to repeat the
-unseal steps (but not the init step) before the CA can become functional again.
+<div class="p-notification--information">
+  <p markdown="1" class="p-notification__response">
+    It is <strong><em>critical </em></strong> that you save all five unseal keys as well as the root
+    token.  If the <strong>Vault</strong> unit is ever rebooted, you will have to repeat the
+    unseal steps (but not the init step) before the CA can become functional
+    again.
+  </p>
+</div>
 
-## Transitioning an existing CDK from EasyRSA to Vault
+## Transitioning an existing cluster from EasyRSA to Vault
 
-An existing **CDK** deployment which is using Vault can easily transition to
-**Vault** simply by following the same steps as above, redeploying the same
-base bundle that you initially deployed on top of the existing deployment
-with the addition of the overlay and then following the steps to unseal
-**Vault**. This will transition all of the components of the cluster to
-the new CA and certificates (this will include restarting all of the worker
-nodes and result in a brief bit of downtime). Once that is complete, you will
-need to re-download the `kubectl` config file, since it contains the certificate
-info for connecting to the cluster.  Once you are satisfied with the state of
-the cluster, you can remove the **EasyRSA** application with:
+An existing **Charmed Kubernetes** deployment which is using EasyRSA can
+be transitioned to use **Vault** as a CA.
 
-```bash
-juju remove-application easyrsa
-```
+Folow the same steps outlined above, and redeploy the same
+base bundle which was initially deployed on top of the existing deployment,
+adding the overlay ([download][k8s-vault-yaml]).
 
 <div class="p-notification--information">
   <p markdown="1" class="p-notification__response">
     <span class="p-notification__status">Note:</span>
     Using an unpinned bundle revision may result in the cluster charms being
     upgraded. To avoid this, specify the exact bundle revision you deployed
-    with originally, or use `juju export-bundle` to create a custom bundle
-    based on exactly what you currently have deployed.
+    originally, or use `juju export-bundle` to create a custom bundle
+    based on exactly what is currently deployed.
   </p>
 </div>
 
+For example, for a 1.15 release:
+
+```bash
+juju deploy charmed-kubernetes-139 --overlay ./k8s-vault.yaml
+```
+
+Then follow the steps to unseal **Vault**. This will transition all of the
+components of the cluster to the new CA and certificates
+(this will include restarting all of the worker
+nodes and result in a brief bit of downtime).
```

Comment:
This should be clarified to only affect the control place.  Maybe something like, "(this will include restarting all of the master and worker nodes; the workloads will be unaffected, but you will be unable to issue cluster commands with `kubectl` for a brief period)"

Reply:
This, or just remove this bit entirely.